typos.toml

```diff
@@ -0,0 +1,2 @@
+[files]
+extend-exclude = ["scripts/*.sh"]
```

Comment:
Why excludes `scripts/*.sh`?

Reply:
It's checking `set -ue` as a typo.

Reply:
Maybe maintain an allow-list is a better way https://github.com/marketplace/actions/typos-action#false-positives

Reply:
Yes, use `default.extend-words` instead of whole file.

Reply:
@dongxuwang Can we use `.typos.toml`? I think the hidden files will be better?